Worn Out/Worn Out/View Controllers/ShoeDetailTableViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  ShoeDetailTableViewController.swift
+//  Worn Out
+//
+//  Created by Vici Shaweddy on 3/4/20.
+//  Copyright © 2020 Vici Shaweddy. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+class ShoeDetailTableViewController: UITableViewController {
+    @IBOutlet weak var totalMiles: UILabel!
+    @IBOutlet weak var nicknameLabel: UILabel!
+    @IBOutlet weak var brandLabel: UILabel!
+    
+    private let context = CoreDataStack.shared.mainContext
+    
+    var shoe: Shoe?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.updateViews()
+        self.tableView.dataSource = self
+        self.tableView.delegate = self
+        self.tableView.register(RunListHeaderView.self, forHeaderFooterViewReuseIdentifier:"RunListHeader")
```

Comment:
You might want to use a private constant for this, so you can be sure you didn't miss-type it in the future.

Reply:
I am going to refactor everything at the end. Thanks!